apps/server/src/modules/board/repo/recursive-delete.vistor.ts

```diff
@@ -65,6 +65,7 @@ export class RecursiveDeleteVisitor implements BoardCompositeVisitorAsync {
 
 	async visitDrawingElementAsync(drawingElement: DrawingElement): Promise<void> {
 		await this.drawingElementAdapterService.deleteDrawingBinData(drawingElement.id);
+		await this.filesStorageClientAdapterService.deleteFilesOfParent(drawingElement.id);
```

Comment:
What is mean with drawingElement? Is this not a tldrawElement?  Do you want to make it replaceable without changing code?
If we speak than it make sense that you show me short how the data structure look and how it is embedded in vue.

Reply:
DrawingElement is just a boardnode with type: 'drawing-element'. I didn't choose the naming. I guess it doesn't matter what kind of drawing tool sits behind this as on the frontend site its just a card with a link opening a drawing tool in a new tab.